doc/pages/expansions.asciidoc

```diff
@@ -218,6 +218,23 @@ The following expansions are supported (with required context _in italics_):
     _in window scope_ +
     line of the main cursor, the third component of `%val{selection_desc}`
 
+*%val{history}*::
+    _in buffer, window scope_ +
+    the full change history of the buffer, including undo forks, as a
+    _history_id_ followed by one or more history entries. _history_id_ may
+    be different from `%val{history_id}` to capture uncommitted changes, for
+    example from an InsertChar hook. Each entry is
+    `parent committed redo_child modification0 modification1 ...`.
+    _parent_ is the index of the entry's predecessor.  Entry 0, which is the
+    root of the history tree, will always have `-` for _parent_. _committed_
+    is the count of seconds from January 1, 1970 to the creation of the
```

Comment:
Hmm, in the case of parinfer, whenever it fixes parens it will create a peer node with almost certainly the same timestamp.  For things like `:earlier`, we can prefer the node with the highest index.

In this case, it doesn't make sense to select the old node, since parinfer would re-fix it anyway.  I suppose unless we've turned parinfer off because it got confused and broke our parens.

I propose that if we need higher resolution, we add a decimal portion.  This seems compatible enough to do later if needed?